docs/manual-EN/manual-introduction.md

```diff
@@ -0,0 +1,75 @@
+# Nebula Graph R201910_beta User Manual
+
+### About this manual
+
+This is the Nebula Graph User Manual. It documents Nebula Graph R201910_beta. It may include documentation of features of Nebula Graph versions that have not yet been released. For information about which versions have been released, see  Nebula Graph Release Notes.
```

Comment:
It may include documentation of features of Nebula Graph versions that have not yet been released.  --- Don't do this, delete

Reply:
Fixed, and we need release note.